lib/rubocop/cop/layout/multiline_method_call_indentation.rb

```diff
@@ -214,6 +214,18 @@ def operator_rhs?(node, receiver)
           node.operator_method? && node.arguments? &&
             within_node?(receiver, node.first_argument)
         end
+
+        def relative_indentation(lhs, node)
+          indentation = indentation(lhs) + correct_indentation(node)
+          indentation += indentation(node) if node_indentationed?(lhs, node)
+          indentation
+        end
+
+        def node_indentationed?(lhs, node)
```

Comment:
Guess you meant to write `indented?`, but I really can't tell what this means in the context of a node. :-)

Reply:
Renaming at the night was a bad idea)
I meant that the node already has the indentation. Is `node_already_indented` would be better?

Reply:
Outdated